WooCommerce/Classes/Blaze/BlazeLocalNotificationScheduler.swift

```diff
@@ -133,7 +143,7 @@ private extension DefaultBlazeLocalNotificationScheduler {
 
         Task { @MainActor in
             let notification = LocalNotification(scenario: LocalNotification.Scenario.blazeNoCampaignReminder,
-                                                 userInfo: [:])
+                                                 userInfo: [Constants.siteIDKey: siteID])
```

Comment:
Suggestion: I think we can add a unit test to ensure that `siteID` is added to user info.

Reply:
Good catch! Done in 6f58663b52feca4a4131af781738b490d26f4ece.